lib/bgs/base.rb

```diff
@@ -151,7 +151,7 @@ def client
         ssl_cert_file: @ssl_cert_file,
         ssl_ca_cert_file: @ssl_ca_cert,
         open_timeout: 10, # in seconds
-        read_timeout: 600, # in seconds
+        read_timeout: 120, # in seconds
```

Comment:
Is there a way to override this in our application code instead of having to change the gem?

Reply:
Hmm. Looking at the way it's set up, it doesn't look like `@client` or `options` gets passed in so I think this is the right way to do it.

Reply:
Looks like all the inheriting is taking place in this gem. :/